src/Controller/ModelCollector.php

```diff
@@ -34,6 +35,7 @@
 use ContaoCommunityAlliance\DcGeneral\EnvironmentInterface;
 use ContaoCommunityAlliance\DcGeneral\Exception\DcGeneralInvalidArgumentException;
 use ContaoCommunityAlliance\DcGeneral\Exception\DcGeneralRuntimeException;
+use function sprintf;
```

Comment:
PSR-12 requires an empty line here to separate blocks what does the current style say?

Reply:
Nothing. :disappointed: